spec/requests/admin/connected_apps_controller_spec.rb

```diff
@@ -0,0 +1,174 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+RSpec.describe Admin::ConnectedAppsController do
```

Comment:
This is not a controller spec. It's a request spec. You could put the path here but your describe blocks below are a better place for that. Maybe just `RSpec.describe "Admin ConnectedApp Vine" do`?